bin/rule.ml

```diff
@@ -143,7 +145,8 @@ let run (`Setup ()) (`File md_file) (`Section section) (`Direction direction)
     let options =
       List.map (Fmt.to_to_string pp_prelude) prelude @
       List.map (Fmt.to_to_string pp_prelude_str) prelude_str @
-      [Fmt.to_to_string pp_direction direction]
+      [Fmt.to_to_string pp_direction direction] @
+      (match section with Some s -> [Fmt.(to_to_string pp_section) s] | None -> [])
```

Comment:
Adding the `Option.get_value` (or whatever bette name you can come up with) will also be useful here!

Reply:
I changed to write it like syntax. What do you think?

Reply:
Looks good thanks!